server/src/board/storage.rs

```diff
@@ -64,6 +66,9 @@ impl Service {
                 self.users.remove(&payload);
                 Action::Leave { payload }
             }
+            Action::Move { payload } => {
```

Comment:
for actions we don't want to store - what do you think about introducing the default case and just return the action?
```
_ => action,
```